src/esp/bindings/SceneBindings.cpp

```diff
@@ -40,7 +40,8 @@ void initSceneBindings(py::module& m) {
       .value("EMPTY", SceneNodeType::EMPTY)
       .value("SENSOR", SceneNodeType::SENSOR)
       .value("AGENT", SceneNodeType::AGENT)
-      .value("CAMERA", SceneNodeType::CAMERA);
+      .value("CAMERA", SceneNodeType::CAMERA)
+      .value("OBJECT", SceneNodeType::OBJECT);
```

Comment:
Perhaps this PR is not the time for this, but we need to refactor our enums to pascal capitalization from all CAPs, for consistency.  Would be nice to have this done before 2.0 release.